server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -1019,7 +1019,7 @@ public void applyUpdates(TInfo tinfo, long updateID, TKeyExtent tkeyExtent,
     private void flush(UpdateSession us) {
 
       int mutationCount = 0;
-      Map<CommitSession,Mutations> sendables = new HashMap<>();
+      Map<CommitSession,TabletMutations> sendables = new HashMap<>();
```

Comment:
could be `Map<CommitSession, List<Mutation>>` since only mutations are used from sendables.